### PR TITLE
B 436918994

### PR DESCRIPTION
**Pull request by entrpn**
- Fix cross attention padding in splash attention
- Add option to replicate vae.

### CODE REVIEW

**Review by coolkp — DISMISSED** [review]

File: src/maxdiffusion/tests/wan_transformer_test.py [hunk 1]
```diff
@@ -305,11 +307,7 @@ def test_get_qt_provider(self, mock_qt_rule):
     config_fp8.quantization = "fp8"
     provider_fp8 = WanPipeline.get_qt_provider(config_fp8)
     self.assertIsNotNone(provider_fp8)
-    mock_qt_rule.assert_called_once_with(
```
Comment (coolkp) [inline src/maxdiffusion/tests/wan_transformer_test.py:308]: we don't want to undo this change, sanbao fixed the test earlier. There is no attributes rules so was breaking the test.

File: src/maxdiffusion/tests/wan_transformer_test.py [hunk 2]
```diff
@@ -319,17 +317,7 @@ def test_get_qt_provider(self, mock_qt_rule):
     config_fp8_full.quantization_calibration_method = "absmax"
     provider_fp8_full = WanPipeline.get_qt_provider(config_fp8_full)
     self.assertIsNotNone(provider_fp8_full)
-    mock_qt_rule.assert_called_once_with(
-      module_path='.*',  # Apply to all modules
-      weight_qtype=jnp.float8_e4m3fn,
-      act_qtype=jnp.float8_e4m3fn,
-      bwd_qtype=jnp.float8_e5m2,
-      bwd_use_original_residuals=True,
-      disable_channelwise_axes=True, # per_tensor calibration
```
Comment (coolkp) [inline src/maxdiffusion/tests/wan_transformer_test.py:328]: Ditto
